src/ipahealthcheck/core/main.py

```diff
@@ -220,7 +219,7 @@ def main():
 
     for name, registry in find_registries().items():
         try:
-            registry.initialize(framework)
+            registry.initialize(framework, config)
         except Exception as e:
             print("Unable to initialize %s: %s" % (name, e))
             sys.exit(1)
```

Comment:
It also needs this:

diff --git a/src/ipahealthcheck/core/main.py b/src/ipahealthcheck/core/main.py
index 3698c9e..05fac35 100644
--- a/src/ipahealthcheck/core/main.py
+++ b/src/ipahealthcheck/core/main.py
@@ -247,10 +247,10 @@ def main():
         if options.source:
             results = limit_results(results, options.source, options.check)
     else:
-        results, available = run_service_plugins(plugins, config,
+        results, available = run_service_plugins(plugins,
                                                  options.source,
                                                  options.check)
-        results.extend(run_plugins(plugins, config, available,
+        results.extend(run_plugins(plugins, available,
                                    options.source, options.check))
 
     if options.source and len(results.results) == 0:

Reply:
Done